app/persistence/entity/fee.go

```diff
@@ -25,4 +25,5 @@ type Fee struct {
 	Status        string
 	TransferID    sql.NullString
 	BurnEventID   sql.NullString
+	LockEventID   sql.NullString
```

Comment:
Fees for `lock` events will be distributed in the smart contract and I am not sure that we will store any of this in our DB